### PR TITLE
Update classification-of-standard-query-operators-by-manner-of-execut…

### PR DESCRIPTION
**Pull request by poojarsn**
…ion.md

Updated deferred and Immediate section only.

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

### CODE REVIEW

File: docs/csharp/programming-guide/concepts/linq/classification-of-standard-query-operators-by-manner-of-execution.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ The LINQ to Objects implementations of the standard query operator methods execu
   
 ### Immediate  
 
- Immediate execution means that the data source is read and the operation is performed at the point in the code where the query is declared. All the standard query operators that return a single, non-enumerable result execute immediately.  
+ Immediate execution means that the data source is read and the operation is performed at the point in the code where the query is declared. All the standard query operators that return a single, non-enumerable result execute immediately. You can force a query to execute immediately of by calling ToList, ToArray methods. Immediate execution doesn't provide the facility of query re-usability, since it always contains the same data which is fetched at the time of query declaration.
```
Comment (BillWagner) [inline docs/csharp/programming-guide/concepts/linq/classification-of-standard-query-operators-by-manner-of-execution.md:15]: A couple nits, and replace method names with linkable xrefs:

```suggestion
 Immediate execution means that the data source is read and the operation is performed once. All the standard query operators that return a scalar result execute immediately. You can force a query to execute immediately using the <xref:System.Linq.Enumerable.ToList%2A?displayProperty=nameWithType> or <xref:System.Linq.Enumerable.ToArray%2A?displayProperty=nameWithType> methods. Immediate execution provides reuse of query results, not query declaration. The results are retrieved once, then stored for future use.
```

**Review by BillWagner — CHANGES_REQUESTED** [review]
Thanks for making these changes @poojarsn 

I had a couple suggestions to improve the clarity. Once you accept those, this is ready to merge.

Thanks for helping us improve.

**Comment by BillWagner** [comment]
Hi @poojarsn

Did you close this by mistake? It looks like you accepted my changes, and I was ready to merge this.

Let me know, and we'll take care of it.

**Comment by poojarsn** [comment]
Yes, I happened to close by mistake. I would be grateful if you can merge
it. Thank you much.

**Review by BillWagner — APPROVED** [review]

**Pull request merged**